OneSignalSDK/unittest/src/test/java/com/test/onesignal/NotificationOpenedActivityHMSIntegrationTestsRunner.java

```diff
@@ -102,18 +127,34 @@ public void emptyIntent_doesNotThrow() {
 
     @Test
     public void barebonesOSPayload_startsMainActivity() throws Exception {
-        helper_initSDKAndFireHMSNotificationOpenIntent();
+        helper_initSDKAndFireHMSNotificationBarebonesOSOpenIntent();
 
         Intent startedActivity = shadowOf(RuntimeEnvironment.application).getNextStartedActivity();
         assertEquals(startedActivity.getComponent().getClassName(), BlankActivity.class.getName());
     }
 
     @Test
     public void barebonesOSPayload_makesNotificationOpenRequest() throws Exception {
-        helper_initSDKAndFireHMSNotificationOpenIntent();
+        helper_initSDKAndFireHMSNotificationBarebonesOSOpenIntent();
         assertNotificationOpenAtIndex(1, UserState.DEVICE_TYPE_HUAWEI);
     }
 
+    private static final String TEST_ACTION_ID = "myTestActionId";

```

Comment:
can this go in the top with the other constants? 😁

Reply:
Done